index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <h1 id="title">freeCodeCamp Survey Form</h1>
+  <p id="description">freeCodeCamp Survey Form</p>
+  <form id="survey-form">
+    <label id="name-label" for="name">Name</label>
+    <input type="text" id="name" placeholder="John Doe" required>
+    <label id="email-label" for="email">Email</label>
+    <input type="email" id="email" placeholder="email@email.com" required>
+    <label id="number-label" for="number">Age</label>
+    <input type="number" id="number" min="21" max="90" placeholder="Ex: 25">
```

Comment:
[Optional] In the long term, adding a class to certain objects like p and label could be useful for better customization.